src/main/webapp/scripts/post-comment.js

```diff
@@ -0,0 +1,27 @@
+function postVideoComment(videoId, commentText) {
+    return new Promise((resolve, reject) => {
+        gapi.client.request(buildRequestBody(videoId, commentText))
+            .then(res => res.json)
+            .then(data => resolve(data))
+            .catch(err => reject(err.result.error.message));
+    });
+}
+
+function buildRequestBody(videoId, commentContent) {
+    return {
+        'path': '/youtube/v3/commentThreads?part=snippet',
+        'method': 'POST',
+        'body': {
+            "snippet": {
```

Comment:
nit: please use single or double quotes consistently here and throughout the code

Reply:
Fixed, nice catch :D